packages/amplication-client/src/VersionControl/CommitResourceListItem.scss

```diff
@@ -0,0 +1,39 @@
+@import "../style/index.scss";
+
+.commit-resource-list-item {
+  &__title-row {
+    @include regular-10;
+
+    &__resource-icon {
+      .amp-icon {
+        font-size: 12px !important;
+      }
+    }
+
+    &__title {
+      @include regular-12;
+      @include semiBold;
+      color: #ffffff;
+    }
+  }
+
+  &__description-row {
+    @include regular-10;
+
+    .amp-text {
+      font-size: 10px;
+      line-height: 15px;
```

Comment:
use variables

Reply:
ok sure, can i convert it to draft and mix it or create new pr with the fix.